activestorage/reductionist.py

```diff
@@ -144,12 +157,11 @@ def build_request_data(source: str, bucket: str, object: str, offset: int,
     return {k: v for k, v in request_data.items() if v is not None}
 
 
-def request(url: str, username: str, password: str, request_data: dict):
+def request(session: requests.Session, url: str, request_data: dict):
     """Make a request to a Reductionist API."""
-    response = requests.post(
+    response = session.post(
         url,
         json=request_data,
-        auth=(username, password)
```

Comment:
missed this one: will this be a problem when we get out of Minio and start using the actual AWS S3?

Reply:
No functional change here - the auth parameters are moved from individual requests to the session.

Reply:
got it, thanks! :+1: